app/scripts/models/account.js

```diff
@@ -360,11 +358,7 @@ define([
           .then(function (client) {
             profileClient = client;
             var accessToken = self.get('accessToken');
-            if (accessToken) {
```

Comment:
Are we guaranteed to have an `accessToken` here? These checks were added to eliminate the 4xx errors we saw on the profile server b/c of calls w/o the `accessToken`.